spec/kahuna_client/client/adaptive_campaign_spec.rb

```diff
@@ -0,0 +1,47 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
otherwise can change `.rspec` file with require spec_helper flag so we don't need this at all in every spec file